core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -273,7 +272,7 @@ private[log] class LogCleanerManager(val logDirs: Seq[File],
    *  6. If the partition is already paused, a new call to this function
    *     will increase the paused count by one.
    */
-  def abortAndPauseCleaning(topicPartition: TopicPartition): Unit = {
+  def abortAndPauseCleaning(topicPartition: TopicPartition, partitionDeleted: Boolean = false): Unit = {
```

Comment:
Actually, we already have `abortCleaning` for this purpose and `abortCleaning` calls `abortAndPauseCleaning`. I was trying to avoid logging a message when the partition is deleted because it does not bring much and literally flood the log when many partitions are deleted.

While re-looking at this, I have done it differently now. I have found that logs were spread between the LogCleanerManager and the LogManager and that we were logging when resuming cleaning but not all the time. I have consolidated all the cases where we want to log explicitly in helper methods. It also helps with not always having to check if `cleaner != null`. Let me know what you think.